openlibrary/templates/type/list/view_body.html

```diff
@@ -272,4 +273,4 @@ <h3 class="collapse black uppercase">$label</h3>
 
 $if meta_cover_url is not None:
     $add_metatag(property="og:image", content=meta_cover_url)
-    $add_metatag(property="twitter:image", content=meta_cover_url)
+    $add_metatag(property="twitter:image", content="https://openlibrary.org/$(list.key)/preview.png")
```

Comment:
We may want to use whatever host is specified by the server rather than hard-coding openlibrary.org.

If the code is on testing.openlibrary.org but the url is hard-coded as openlibrary.org then the url won't resolve correctly.

Reply:
I think there may be a variable `request.domain` which we can use instead